src/main/fc/core.c

```diff
@@ -404,6 +396,14 @@ void tryArm(void)
             }
             return;
         }
+
+#ifdef USE_DSHOT_TELEMETRY
+        // send the command to enable DSHOT telemetry
+        if (isMotorProtocolDshot() && motorConfig()->dev.useDshotTelemetry) {
+            pwmWriteDshotCommand(255, getMotorCount(), DSHOT_CMD_SIGNAL_LINE_CONTINUOUS_ERPM_TELEMETRY, true);
```

Comment:
I think this should be switched to use queued Dshot commands - queued Dshot commands were introduced because the use if blocking Dshot commands to enable crash flip mode when arming led to sporadic malfunctions when the command was sent too shortly after the last motor update had been sent, causing the ESC to miss the command while it was still busy processing the motor update. The same will likely happen sporadically for the commands to enable / disable Dshot telemetry.

As the queue depth for Dshot commands is only 1 deep at the moment, this will likely also need a change to Dshot command queuing.

Blocking Dshot commands should really only be used in the way they are used in CLI, where motor updates are disabled, and then, after a delay, Dshot commands are sent.